FirebaseFunctions/Sources/Functions.swift

```diff
@@ -25,6 +24,9 @@ import FirebaseSharedSwift
   import GTMSessionFetcherCore
 #endif
 
+// Avoids exposing internal FirebaseCore APIs to Swift users.
+@_implementationOnly import FirebaseCoreExtension
```

Comment:
Awesome!

Reply:
The leading underscore makes me nervous that it'll break at some point unexpectedly... but it's better than the alternative at least 🤷 .